src/build.py

```diff
@@ -195,6 +195,10 @@ def NodePlatformName():
     os.path.join(BINARYEN_SRC_DIR, 'test',
                  's2wasm_known_binaryen_shell_test_failures.txt')]
 
+# Optimization levels
+BARE_TEST_OPT_FLAGS = ['O0', 'O2']
+BINARYEN_TEST_OPT_FLAGS = ['O0', 'O3']
```

Comment:
Done. Actually I was torn between the two as well and chose `BINARYEN` to be consistent with the function name `CompileLLVMTortureBinaryen`.